apps/shop/models.py

```diff
@@ -95,3 +96,41 @@ class ShopContactsModel(models.Model):
     email = models.CharField(max_length=100)
     viber_link = models.CharField(max_length=100)
     telegram_link = models.CharField(max_length=100)
+
+User = get_user_model()
+class Order(models.Model):
+    STATUS_CHOICES = [
+        ('new', 'Нове замовлення'),
+        ('confirmed', 'Підтверджено'),
+        ('shipped', 'Відправлено'),
+        ('delivered', 'Доставлено'),
+        ('complete', 'Виконано'),
+        ('canceled', 'Скасовано'),
+    ]
+    PAYMENT_CHOICES = [
+        ('online', 'Оплата онлайн'),
+        ('cod', 'Накладений платіж'),
+        ('postpay', ' После оплата')
+    ]
+
+    first_name = models.CharField(max_length=50)
+    last_name = models.CharField(max_length=50)
+    country = models.CharField(max_length=100)
+    region = models.CharField(max_length=100)
+    city = models.CharField(max_length=50)
+    delivery_service = models.CharField(max_length=100)
+    postoffice = models.CharField(max_length=50)
+    postbox = models.CharField(max_length=50)
```

Comment:
both can be None, because you cant deliver to postoffice and postbox at the same time, one every time is None